synapse/models/dns.py

```diff
@@ -23,20 +29,21 @@ def _normPyStr(self, valu):
         # Break out fqdn / ipv4 / ipv6 subs :D
         subs = {}
         # ipv4
-        inarpa = '.in-addr.arpa'
-        inarpa6 = '.ip6.arpa'
-        if norm.endswith(inarpa):
+        if intre.match(norm):
```

Comment:
We should use `norm.isnumeric()` here for performance

Reply:
fixed